Animations/Css Animations/Examples/custom.css

```diff
@@ -0,0 +1,87 @@
+* {
+	margin: 0;
+	padding: 0;
+	border: 0;
+}
+h1 {
+	font-size: 30px;
+	line-height: 32px;
+	color: blue;
+}
+h3 {
+	font-size: 20px;
+	line-height: 22px;
+	margin: 20px;
+}
+p {
+	font-size: 18px;
+	line-height: 20px;
+	margin-left:50%;
+}
+.outerBox {
+	margin-bottom: 20px;
+}
+
+/*fadeInLeft*/
+
+.fadeInLeft {
+	animation:fadeInLeft 1s ease-in-out 0.5s;
+}
+@keyframes fadeInLeft { 
+	0% {
+		opacity: 0;
+		transform: translate(-100%,0);
+	}
+	100% {
+		opacity: 1;
+		transform: none;
+	}
+}
+
+/*FadeInRight*/
+
+.fadeInRight {
+	animation:fadeInRight 1s ease-in-out 0.5s 2;
+}
+@keyframes fadeInRight {
```

Comment:
use **@-webkit-** for safari version support (4.0 - 8.0).

Reply:
OK

Reply:
Done